entrypoint

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+event = os.environ.get('EVENT_NAME', None)
+if event == "pull_request_target":
+    os.system("enarx-pr-request")
+    os.system("enarx-pr-merge")
+    os.system("enarx-pr-assign")
```

Comment:
Doing this will very likely cause us to exceed the GH API rate limiting. It would be better to port one single action to efficiently use the input data.